website/content/docs/concepts/acl.mdx

```diff
@@ -0,0 +1,85 @@
+---
+layout: docs
+page_title: Access Control List (ACL)
+description: Learn about Nomad's ACL subsystem
+---
+
+# Access Control List (ACL)
+
+The Nomad ACL system is designed to be intuitive, high-performance, and to
+provide administrative insight. At the highest level, there are four major
+components to the ACL system: tokens, policies, roles, and capabilities. The
+two components named auth methods and binding rules are specific to Nomad's
+Single Sign-On (SSO) ACL capabilities.
+
+The Nomad [access control tutorials] provide detailed information and guidance
+on Nomad ACL system.
+
+### Policy
+
+Policies consist of a set of rules defining the capabilities or actions to be
+granted. For example, a `readonly` policy might only grant the ability to list
+and inspect running jobs, but not to submit new ones. No permissions are
+granted by default, making Nomad a default-deny system.
+
+Each policy comprises one or more rules. The rules define the capabilities of a
+Nomad ACL token for accessing the objects in a Nomad cluster, objects like
+namespaces, node, agent, operator, quota, etc.
```

Comment:
Maybe cross link here to [ACL Policy spec](https://developer.hashicorp.com/nomad/docs/other-specifications/acl-policy)? Something like "For more information on writing policies, see the ACL policy reference doc"?